go.mod

```diff
@@ -13,6 +13,8 @@ replace (
 // Nomad is built using the current source of the API module.
 replace github.com/hashicorp/nomad/api => ./api
 
+replace github.com/hashicorp/nomad/client => ./client
+
```

Comment:
Did you mean to commit this? `./api` is only replaced because it is also a Go module while `./client` is just a Go package.

Reply:
Something was breaking when I was trying to build. Not sure what this was about. I'll remove it.